projects/acs-engine/azure-purge/git-secret.yaml

```diff
@@ -0,0 +1,10 @@
+### to get this secret:
```

Comment:
Why is this needed?

Reply:
we build from azure-misc, which is private. so we need to have ssh key to access it (easiest way) or move some of the stuff to public repo

Reply:
I propose to use ssh key I just added to misc repo and add it as deploy key to only our repo.